pkgs/applications/radio/yaac/default.nix

```diff
@@ -0,0 +1,34 @@
+{ makeWrapper, lib, stdenv, fetchurl, unzip, jre8 }:
```

Comment:
One item per line is preferable for future diff.

Reply:
Updated in latest.